templates/songs.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang=en>
+<head>
+<meta http-equiv=content-type content="text/html; charset=utf-8" />
+<meta name=viewport content="width=device-width, initial-scale=1">
+<link rel=icon type=image/x-icon href=/static/hacky-day.png />
+<link rel=stylesheet type=text/css href=/static/index.css />
+
+<title>hack·star - Song Management</title>
+<script>
+function deleteSong(songId) {
+  if (confirm('Are you sure you want to delete this song? This action cannot be undone.')) {
+    fetch('/songs/' + songId + '/delete', {
+      method: 'DELETE'
+    }).then(response => {
+      if (response.ok) {
+        location.reload();
+      } else {
+        alert('Error deleting song');
+      }
+    }).catch(error => {
+      alert('Error deleting song: ' + error);
+    });
+  }
+}
+</script>
+</head>
+<body>
+
+	<header class=top>
+		<a href=/><img src=/static/hacky-day.png alt='hacky·day' /></a>
+    Song Management
+	</header>
+
+	<main>
+    <div id="songs-container">
+      <h2>Songs ({{ songs|length }})</h2>
+      
+      {% if songs %}
+      <div id="songs-content">
+        {% for song in songs %}
+        <div class="song-item">
+          <div class="song-metadata">
+            {% if song.cover %}
+            <div class="song-cover">
+              <img src="{{ song.cover }}" alt="Cover" />
+            </div>
+            {% endif %}
+            <div class="song-info">
+              <div class="song-title">{{ song.title }}</div>
+              <div class="song-artist">{{ song.artist }}</div>
+              <div class="song-year">{{ song.release_date }}</div>
+              <div class="song-id">ID: {{ song.hex_id }}</div>
+            </div>
+          </div>
+          
+          <div class="song-status">
+            <div class="import-status import-status-{{ song.import_status }}">
+              Status: {{ song.import_status }}
+            </div>
+            <div class="import-source">{{ song.import_source }}</div>
+          </div>
+          
+          <div class="song-actions">
+            {% if song.import_status == 'finished' %}
+            <a href="/song/{{ song.hex_id }}" class="song-play-btn" target="_blank">Play</a>
```

Comment:
Instead of this button, can we put a play/pause icon over the cover to play back the song?